lib/mpool/src/mpool.c

```diff
@@ -32,6 +32,11 @@ struct mpool {
     const char          home[]; /* flexible array */
 };
 
+const char *const mpool_mclass_to_string[MP_MED_COUNT] = {
+    [MP_MED_CAPACITY] = MP_MED_NAME_CAPACITY,
+    [MP_MED_STAGING] = MP_MED_NAME_STAGING,
+};
+
```

Comment:
Does using MP_MED_COUNT make this structure bigger than necessary?

Reply:
MP_MED_COUNT is MP_MED_STAGING + 1 where MP_MED_STAGING is 1 and MP_MED_CAPACITY is 0, so no. I was just being explicit about size.